Document/0x06d-Testing-Data-Storage.md

```diff
@@ -415,9 +415,9 @@ A generalized approach to this issue is to use a define to enable `NSLog` statem
 
 Navigate to a screen that displays input fields that take sensitive user information. Two methods apply to checking log files for sensitive data:
 
-1. Connect to the iOS device and execute the following command:
+1. Connect to the iOS device and execute the following command (available through Cydia):
```

Comment:
Thanks! That's a nice altnerative!
l would rather use tail here and have an alternative offered directly below this text in which you introduce ondeviceconsole and how you can get it (so : which repo do you need? how do you configure it?)

Reply:
Soo, I have tail installed on my jailbroken iOS device...

Reply:
Will change it a bit to have both alternatives. Thank you for the suggestion :).